packages/venia-concept/src/RootComponents/Category/category.js

```diff
@@ -88,11 +89,13 @@ const Category = props => {
     }
 
     // Show the loading indicator until data has been fetched.
-    if (!totalPagesFromData) {
+    if (!totalPagesFromData && totalPagesFromData !== 0) {
```

Comment:
Let's check explicitly for `null` here, that'll make the intention of this code more clear 👍

Reply:
Agreed.